### PR TITLE
Upgrade to SilverStripe 4 support

### PR DESCRIPTION
**Pull request by stephenmcm**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-	"name": "silverstripe-australia/mailcapture",
+	"name": "symbiote/mailcapture",
```
Comment (nyeholt) [inline composer.json]: symbiote/silverstripe-mailcapture

File: src/Controller/CapturedEmailController.php [hunk 2]
```diff
@@ -20,7 +27,7 @@ public function view() {
 		$id = (int) $this->getRequest()->param('ID');
 
 		if ($id) {
-			$email = DataList::create('CapturedEmail')->byID($id);
+			$email = DataList::create(CapturedEmail::class)->byID($id);
```
Comment (nyeholt) [inline src/Controller/CapturedEmailController.php]: Preferable to use `CapturedEmail::get()->byID($id);`

File: src/Email/CaptureMailer.php [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                    $return .= "$address";
+                    if ($title) {
+                        $return .= " <$title>";
+                    }
+                    $return .= ", ";
+                }
+                return $return;
+            };
+
+			$mail = new CapturedEmail();
```
Comment (nyeholt) [inline src/Email/CaptureMailer.php]: `CapturedEmail::create();`

**Review by nyeholt — APPROVED** [review]
A few minor things - make the change and good to merge.

File: src/Controller/CapturedEmailController.php (hunk 2 above)
Comment (stephenmcm) [reply src/Controller/CapturedEmailController.php]: Yeah totally, not sure what the original intent there was, this was part of the raw upgrade so I just 'made it work'.

**Pull request merged**